core/sqf/monitor/linux/commaccept.cxx

```diff
@@ -33,6 +33,7 @@ using namespace std;
 #include <signal.h>
 #include <unistd.h>
 
+extern CCommAccept CommAccept;
```

Comment:
There are no dependencies or danger or danger of race conditions, but it is good to remember this.